opentelemetry-sdk/src/opentelemetry/sdk/error_handler/__init__.py

```diff
@@ -61,6 +61,7 @@ def _handle(self, error: Exception, *args, **kwargs):
 
 from abc import ABC, abstractmethod
 from logging import getLogger
+from typing import Optional
```

Comment:
I'd rather prefer to use `from __future__ import annotations` on the top of the imports, and use pipe (`|`) annotation.